pkg/telemetry/marketing/device.go

```diff
@@ -0,0 +1,65 @@
+package marketing
+
+import (
+	"context"
+	"strings"
+
+	"github.com/pkg/errors"
+	"github.com/stackrox/rox/generated/storage"
+	"github.com/stackrox/rox/pkg/env"
+	"github.com/stackrox/rox/pkg/version"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/rest"
+)
+
+const annotation = "rhacs.redhat.com/telemetry-apipaths"
+const orgID = "rhacs.redhat.com/organization-id"
+const sku = "rhacs.redhat.com/sku"
+
+var config *Config
+
+// GetDeviceConfig collects the central instance telemetry configuration.
```

Comment:
What else beyond central id we want to collect and why?

Reply:
We need to have a correlation ID to later join data from other telemetry reporters. This is yet to be defined.